src/business/UserBusiness.ts

```diff
@@ -0,0 +1,107 @@
+import RelationshipDatabase from "../data/RelationshipDatabase";
+import CustomError from "../err/CustomError";
+import User, {
+  DeleteFriendshipDTO,
+  MakeFriendshipDTO,
+  loginDTO,
+  signUpDTO,
+} from "../models/User";
+import UserDatabase from "../data/UserDatabase";
+import HashManager from "../services/HashManager";
+import Authenticator from "../services/Authenticator";
+import IdGenerator from "../services/IdGenerator";
+import RefreshTokenBusiness from "./RefreshTokenBusiness";
+
+export default class UserBusiness {
+  public async deleteFriendship({ friend_id, user_id }: DeleteFriendshipDTO) {
+    if (!friend_id) {
+      throw new CustomError("Please insert a correct Id", 400);
+    }
+
+    const RelationshipDb = new RelationshipDatabase();
+    const userFriends = await RelationshipDb.getFriendById(user_id);
+
+    for (let i = 0; i < userFriends.length; i++) {
+      if (userFriends[i].friend_id === friend_id) {
+        await RelationshipDb.deleteFriendship(user_id, friend_id);
+
+        return;
+      }
+    }
```

Comment:
Evitem o uso do `for(let i = 0;...)`, porque ele é meio confuso! Recomendo o uso do `for(const userFriend of userFriends)`.